examples/C++/DDS/BasicConfigurationExample/BasicConfigurationHelloWorldPublisher.h

```diff
@@ -0,0 +1,145 @@
+// Copyright 2021 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @file BasicConfigurationHelloWorldPublisher.h
+ *
+ */
+
+#ifndef _EPROSIMA_FASTDDS_EXAMPLES_CPP_DDS_BASICCONFIGURATIONEXAMPLE_BASICCONFIGURATIONHELLOWORLDPUBLISHER_H_
+#define _EPROSIMA_FASTDDS_EXAMPLES_CPP_DDS_BASICCONFIGURATIONEXAMPLE_BASICCONFIGURATIONHELLOWORLDPUBLISHER_H_
+
+#include <atomic>
+#include <condition_variable>
+#include <mutex>
+
+#include <fastdds/dds/domain/DomainParticipant.hpp>
+#include <fastdds/dds/publisher/DataWriterListener.hpp>
+#include <fastdds/dds/topic/TypeSupport.hpp>
+
+#include "HelloWorldPubSubTypes.h"
+
+/**
+ * Class used to group into a single working unit a Publisher with a DataWriter, its listener, and a TypeSupport member
+ * corresponding to the HelloWorld datatype
+ */
+class HelloWorldPublisher
+{
+public:
+
+    HelloWorldPublisher();
+
+    virtual ~HelloWorldPublisher();
+
+    //!Initialize the publisher
```

Comment:
Keep the same comment style in the whole example. 
- Space between '//' and comment
- Use //! or // for doxygen comments, but always the same

Reply:
:+1: